### ISSUE TITLE
Performance opportunity when searched file name is exact

### ISSUE THREAD

**Issue by bartoli** [stefankueng/grepWin#274]
In the use case with a very big folder, each subfolder having a file with a given name and lots of other stuff, and we want to find which of those files with known name contains some text. It looks like grepWin is still looking at all files of a folder, and looking if the name of each file equals the searched one.
When a full file name without wildcard is provided, it would be faster to just test in each folder if that name exists in the folder than comparing the name of each file in the folder to the searched name

**Comment by perboe01** [comment]
I agree. I tried to pass a the full name of a file in /filemask  but does not work.
